eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -76,12 +79,14 @@ def enroll_on_program(program_uuid, *arg, **kwargs):
     """
     results = []
     errors = []
+    LOG.info('Enrolling on program: %s', program_uuid)
     try:
         data = get_program(program_uuid)
     except Exception as err:  # pylint: disable=broad-except
         raise APIException(repr(err))
     for course in data['courses']:
         if course['course_runs']:
+            LOG.info('Enrolling on course: %s', course['key'])
```

Comment:
This message should be about the process we are in. So "enrolling on course_run: xxxx" and it should bo after we have run `get_preferred_course_run`